app/presenters/track.rb

```diff
@@ -40,6 +40,14 @@ def trackler_track
         Trackler.tracks[@track_id]
       end
 
+      def better_content(topic_key)
```

Comment:
You're right, that's a confusing name. I renamed it `contributing_instructions` in https://github.com/exercism/exercism.io/pull/3475/commits/a663a3d4d34034a460099efeee36a396125ae46d